dashboard/server.js

```diff
@@ -6,9 +6,8 @@ var config = require("./webpack.config.js");
 var reverseProxyPort = 9001
 var nodeServerPort = 8000
 var cmsPort = 8080
-var vaultPort = 8200
 
-var loadDashboardFromCms = true
+var loadDashboardFromCms = false
```

Comment:
Why this change?  Seems unrelated to the pull request.

Is false the better default setting?  I'm guessing it is better for hot-reload or similar.

Reply:
Yes, that's correct about hot-reload. If you're testing the dashboard, you want Webpack to serve the page so that you don't have to rebuild CMS every time you make a change

Reply:
Yeah, I think it is for hot reloading, else you have to rebuild the jar and restart the webapp every time you adjust a css margin.